README.md

```diff
@@ -48,6 +48,23 @@ LYK_VERBOSE=1 ../src/lua all.lua
 LYK_VERBOSE=1 sh ./test.sh
 ```
 
+## C-vise
```

Comment:
Maybe we should put this part of the README into a new file `creduce/README`, and then instead of "pd" filenames we can use relative filenames e.g. "../src/lua" and similar?

Reply:
updated 👉  0940e4b7173b60a87a0cf5583add4faedba94afc
Some paths need to be absolute (added comment with description) so no much use for "../"